### PR TITLE
Add Sepolia Testnet Integration

### PR DESCRIPTION
**Pull request by kirtiraj22**
### **Description :** 
Fixes : #14 
  
Integrated **Sepolia testnet** into the project

The following changes have been made:
1. Configuration:

- Added Sepolia testnet as an additional network configuration option.
- Integrated the necessary endpoints and settings to connect to the Sepolia blockchain network.
2. Test Ether Allocation:

- Sepolia testnet provides a higher allocation of test ethers per day compared to the existing Goerli testnet.
- Users can now leverage the increased test ether availability for testing and development purposes without the requirement of any ETH on the mainnet.

### **Screenshots :** 
**1. Before :**
- Displayed a transaction notification indicating "Transaction Failed" while using the Sepolia testnet for transactions.

Screenshot (138)

**2. After :**
- Displayed an updated transaction notification indicating "Transaction Complete" after the successful addition of the Sepolia testnet.

Screenshot (136)

Screenshot (137)

### CODE REVIEW

**Review by gurjeetsinghvirdee — COMMENTED** [review]
Edit your PR template and mention the issue number in Top
**`Fixes: #issue number`**

**Comment by kirtiraj22** [comment]
Hey @gurjeetsinghvirdee ,
I have updated the PR so kindly merge it!

**Comment by gurjeetsinghvirdee** [comment]
Don't mention issue number twice and Write like this 
**Fixes: #77**

**Comment by kirtiraj22** [comment]
Hey @gurjeetsinghvirdee , 
Updated the PR and will take care of this next time! 👍

**Review by gurjeetsinghvirdee — APPROVED** [review]

**Comment by gurjeetsinghvirdee** [comment]
Your branch is 7 commit behind  
Sync it 
@kirtiraj22

File: backend/hardhat.config.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ import "hardhat-deploy";
 const GOERLI_RPC_URL = process.env.GOERLI_RPC_URL || "https://eth-goerli.alchemyapi.io/v2/your-api-key"
 const PRIVATE_KEY = process.env.PRIVATE_KEY || "0x"
 const ETHERSCAN_API_KEY = process.env.ETHERSCAN_API_KEY || ""
+const SEPOLIA_RPC_URL = process.env.SEPOLIA_RPC_URL || "https://eth-goerli.alchemyapi.io/v2/your-api-key"
```
Comment (J0SAL) [inline backend/hardhat.config.ts:13]: change the like to "https://eth-sepolia.g.alchemy.com/v2/your-api-key"

File: frontend/constants/abi.json [hunk 2]
```diff
@@ -1 +1 @@
...","name":"_id"},{"type":"uint256","name":"_amount"},{"type":"string","name":"_description"},{"type":"string","name":"_date"},{"type":"string","name":"_category"}],"outputs":[]},{"type":"function","name":"addIncome","constant":false,"payable":false,"gas":29000000,"inputs":[{"type":"string","name":"_id"},{"type":"uint256","name":"_amount"},{"type":"string","name":"_description"},{"type":"string","name":"_date"},{"type":"string","name":"_category"}],"outputs":[]},{"type":"function","name":"deleteTransaction","constant":false,"payable":false,"gas":29000000,"inputs":[{"type":"string","name":"id"}],"outputs":[]},{"type":"function","name":"getUserTransactions","constant":true,"stateMutability":"view","payable":false,"gas":29000000,"inputs":[],"outputs":[{"type":"tuple[]","components":[{"type":"string","name":"id"},{"type":"uint8","name":"ttype"},{"type":"string","name":"category"},{"type":"string","name":"description"},{"type":"uint256","name":"amount"},{"type":"string","name":"date"},{"type":"bool","name":"is_deleted"}]}]},{"type":"function","name":"getUserTransactionsLen","constant":true,"stateMutability":"view","payable":false,"gas":29000000,"inputs":[],"outputs":[{"type":"uint256"}]}]
```
Comment (J0SAL) [inline frontend/constants/abi.json:1]: updo the changes to this file, I don't think Abi should change w.r.t network

**Review by J0SAL — CHANGES_REQUESTED** [review]

**Pull request closed without merging**